docs/tutorials/python/tutorial_scripts/wiki.py

```diff
@@ -0,0 +1,284 @@
+markdown_file_path = "path/to/your_markdown_file.md"
+root_wiki_page = WikiPage(
+    owner_id=my_test_project.id,
+    title="My First Root Wiki Page Version with existing markdown file",
+    markdown=markdown_file_path,
+).store()
+
+# Create a new wiki page with updated content
+root_wiki_page_new = WikiPage(
+    owner_id=my_test_project.id,
+    title="My First Root Wiki Page NEW",
+    markdown="# Welcome to My Root Wiki NEW\n\nThis is a sample root wiki page created with the Synapse client.",
+    id=root_wiki_page.id,
+).store()
+
+# Restore the wiki page to the original version
+wiki_page_restored = WikiPage(
+    owner_id=my_test_project.id, id=root_wiki_page.id, wiki_version="0"
+).restore()
+
+# check if the content is restored
+comparisons = [
+    root_wiki_page.markdown_file_handle_id
+    == wiki_page_restored.markdown_file_handle_id,
+    root_wiki_page.id == wiki_page_restored.id,
+    root_wiki_page.title == wiki_page_restored.title,
+]
+print(f"All fields match: {all(comparisons)}")
+
+# Create a sub-wiki page
+sub_wiki_1 = WikiPage(
+    owner_id=my_test_project.id,
+    title="Sub Wiki Page 1",
+    parent_id=root_wiki_page.id,  # Use the ID of the parent wiki page we created '633033'
+    markdown="# Sub Page 1\n\nThis is a sub-page of another wiki.",
+).store()
+
+# Get an existing wiki page for the project, now you can see one root wiki page and one sub-wiki page
+wiki_header_tree = WikiHeader.get(owner_id=my_test_project.id)
+print(wiki_header_tree)
+
+# Once you know the wiki page id, you can retrieve the wiki page with the id
+retrieved_wiki = WikiPage(owner_id=my_test_project.id, id=sub_wiki_1.id).get()
+print(f"Retrieved wiki page with title: {retrieved_wiki.title}")
+
+# Or you can retrieve the wiki page with the title
+retrieved_wiki = WikiPage(owner_id=my_test_project.id, title=sub_wiki_1.title).get()
+print(f"Retrieved wiki page with title: {retrieved_wiki.title}")
+
+# Check if the retrieved wiki page is the same as the original wiki page
+comparisons = [
+    sub_wiki_1.markdown_file_handle_id == retrieved_wiki.markdown_file_handle_id,
+    sub_wiki_1.id == retrieved_wiki.id,
+    sub_wiki_1.title == retrieved_wiki.title,
+]
+print(f"All fields match: {all(comparisons)}")
+
+# Section 2: WikiPage Markdown Operations
+# Create wiki page from markdown text
+markdown_content = """# Sample Markdown Content
+
+## Section 1
+This is a sample markdown file with multiple sections.
+
+## Section 2
+- List item 1
+- List item 2
+- List item 3
+
+## Code Example
+```python
+def hello_world():
+    print("Hello, World!")
+```
+"""
+
+# Create wiki page from markdown text
+sub_wiki_2 = WikiPage(
+    owner_id=my_test_project.id,
+    parent_id=root_wiki_page.id,
+    title="Sub Page 2 created from markdown text",
+    markdown=markdown_content,
+).store()
+
+# Create a wiki page from a markdown file
+# Create a temporary markdown gzipped file from the markdown_content
+markdown_file_path = "temp_markdown_file.md.gz"
```

Comment:
In most of the tutorials we have been using the "~/temp` directory to write files to